ServerCore/Helpers/LiveEventHelper.cs

```diff
@@ -22,6 +20,17 @@ public static async Task TriggerNotifications(PuzzleServerContext context, Event
             await EventClosingReminder(context, currentEvent, hubContext);
         }
 
+        public static List<LiveEventSchedule> GetTeamSchedule(PuzzleServerContext context, Event currentEvent, Team team)
```

Comment:
Should be async, right?

Reply:
Also, since we have like no time to test this, can you please add a ripcord so we can pull it if necessary? See the [NotMapped] EphemeralHack* properties at the bottom of Event.cs

Reply:
Changes pushed.